src/Scene/Quadtree.js

```diff
@@ -69,67 +69,18 @@ define('Scene/Quadtree', [
     };
 
     /**
-     * @documentation: subdivise node if necessary
+     * @documentation: returns bounding boxes of a node's quadtree subdivision
      * @param {type} node
-     * @returns {Array} four bounding box
+     * @returns {Array} an array of four bounding boxex
      */
-    Quadtree.prototype.up = function(node) {
-        if (node.pendingSubdivision) {
-            return;
-        }
-        if (!this.update(node)) {
-            return;
+    Quadtree.prototype.subdivide = function (node) {
```

Comment:
Naming is confusing: a reasonnable expectation when calling `quadtree.subdivide` is that `quadtree` will be subdivided.
Maybe: `computeNodeSubdivisionBB`?